css/_videolayout_default.scss

```diff
@@ -603,4 +602,39 @@
 
 .moveToCorner + .moveToCorner {
     right: 80px;
+}
+
+.filmstripToolbar {
+    width: 20px;
+    position: absolute;
+    right: 4px;
+    bottom: 20px;
+    z-index: 6;
+
+    button {
+        font-size: 14px;
+        line-height: 1.2;
+        text-align: center;
+        background: transparent;
+        opacity: 0.7;
+        height: auto;
+        width: 100%;
+        padding: 0;
+        margin: 0 1px;
+        border: none;
+
+        -webkit-appearance: none;
+
+        &:hover {
+            opacity: 1;
+        }
+
+        i {
+            cursor: pointer;
+        }
+    }
+}
+
+.filmstripToolbar + #remoteVideos {
```

Comment:
Let's add some comments here for people not so familiar with css selectors. In general we need to comment our css code as much as everything else :)